flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/JobLeaderIdService.java

```diff
@@ -33,7 +33,7 @@
 public interface JobLeaderIdService {
 
     /**
-     * Start the service with the given job leader actions.
+     * Start the service with the given job leader id actions.
```

Comment:
@liuzhuang2017 Thanks for opening the PR, but I don't think this is (much of) an improvement over what's already there.

Reply:
@MartijnVisser Thank you for your reply, I will read this official document carefully.